query/query_functions.py

```diff
@@ -8,3 +8,10 @@
 class QueryFunctions(CBRestConnection):
     def __init__(self):
         super(QueryFunctions).__init__()
+
+    def execute_query_statement(self, query, timeout=None):
+
```

Comment:
Can we add a comment with signature + doc reference. Something like,
        """
        GET :: /pools/default/buckets
        docs.couchbase.com/server/current/rest-api/rest-buckets-summary.html
        """